evoting/lib/election.go

```diff
@@ -34,9 +34,11 @@ type Election struct {
 	Key    kyber.Point           // Key is the DKG public key.
 	Stage  uint32                // Stage indicates the phase of the election.
 
-	Data        []byte // Data can hold auxiliary information.
-	Description string // Description in string format.
-	End         string // End (termination) date.
+	Candidates  []uint32 // Candidates is the list of candidate scipers.
+	MaxChoices  uint32   // MaxChoices is the max votes in allowed in a ballot.
```

Comment:
it's nice to have this as `uint32`, but for convenience I would put this as `int`. Else you have to always cast it when you compare lengths.

Except if @jeffallen says that this has to be `uint32` ;) But then I guess we'll have a ton of other places where it should be `uint32` and not `int`...

Reply:
Again, the Go way is, "int unless something about the data type actually requires that you tell the exact number of bits". When crossing systems, from Go to JS or from Go to the kernel, this is sometimes when you should use explicit numbers of bits.

Is that helpful?

Reply:
+1 to saving some casts while counting in future. I'll set it as `int32` in the protobuf, and since values are in the range [0-9] we won't lose any info in Go either.